zerver/webhooks/travis/doc.md

```diff
@@ -0,0 +1,25 @@
+See your Travis CI build notifications in Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Finally, set up a webhook in your `.travis.yml` file:
+
+```
+notifications:
+webhooks:
+- {{ external_api_uri_subdomain }}/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
+```
+
+By default, pull request events are ignored since most people
+don't want notifications for new pushes to pull requests.  To
+enable notifications for pull request builds, just
+add `&ignore_pull_requests=false` at the end of the url.
+
+
+**Congratulations! You're done!**
+
+When a code build occurs,
+you will receive a notification on the specified stream and topic.
+![](/static/images/integrations/travis/001.png)
```

Comment:
Now that I think about it, lines such as `When a code build occurs...` don't really convey a lot of meaning that isn't implicitly conveyed by the `congrats.md` macro. For future reference, in the interest of brevity and consistency, I would recommend getting rid of lines like that altogether, unless they convey information that's crucial or too specific. :)